tests/ekomark/test_apply.py

```diff
@@ -11,7 +11,7 @@ def test_apply(self, fake_legacy, fake_pdf):
         pdf_grid = apply.apply_pdf(o, fake_pdf)
         assert len(pdf_grid) == len(fake_card["Q2grid"])
         pdfs = pdf_grid[q2_out]["pdfs"]
-        assert list(pdfs.keys()) == o.rotations.targetpids
+        assert list(pdfs.keys()) == list(o.rotations.targetpids)
```

Comment:
Just in case, you can also assert they are the same despite the iterables even without converting both to lists:
```suggestion
        assert all(x == y for x, y in zip(pdfs.keys(), o.rotations.targetpids))
```

No special advantage here (you save some memory and time, but it is definitely negligible here).

Reply:
I take it back: it has the disadvantage of not checking all the elements in the longest iterable, in case the length is different, so you'd need to split into a separate check.
```suggestion
        assert len(pdfs.keys()) == len(o.rotations.targetpids))
        assert all(x == y for x, y in zip(pdfs.keys(), o.rotations.targetpids))
```

Either we write a function for this, to consistently use, or it is only more cumbersome (for a negligible advantage).

Reply:
Once more (I believe I told you somewhere else), `.keys()` is not required when you iterate on it, and `list()` is iterating.

Reply:
Yes i know but for the moment I wanted to keep the changes to the tests as minimal as possible. I will drop `.keys` now. :)